resources/lang/es/layout.php

```diff
@@ -54,14 +54,6 @@
             '_' => 'beatmapsets',
             'discussion' => 'modding',
         ],
-        'ranking' => [
```

Comment:
should rename this to `rankings`, rename `getCharts` to `charts`, keep `_`, and remove everything else.